src/hotspot/share/gc/g1/g1YoungGCPostEvacuateTasks.cpp

```diff
@@ -109,7 +111,7 @@ class G1PostEvacuateCollectionSetCleanupTask1::RemoveSelfForwardPtrsTask : publi
 
   double worker_cost() const override {
     assert(_evac_failure_regions->evacuation_failed(), "Should not call this if not executed");
-    return _evac_failure_regions->num_regions_failed_evacuation();
+    return _evac_failure_regions->num_regions_failed_evacuation() * G1YoungGCEvacFailureInjector::evacuation_failure_worker_cost();
```

Comment:
This should not depend on anything in the evacuation failure injector. That seems to be the wrong type of dependency. Why not just use the global here?